examples/runc-hook/README.md

```diff
@@ -0,0 +1,31 @@
+# Example with the runcfanotify package
+
+This example uses the runcfanotify package
+("github.com/kinvolk/inspektor-gadget/pkg/runcfanotify") in order to be execute
+custom OCI hooks.
```

Comment:
Is this outdated? This is not changing the config.json to use OCI hooks, right? This is just running a hook by tricks with fanotify, OCI hooks (while similar on what they can achieve) are not used at all IIUC